src/test/java/com/qalabs/seleniumbasics/spotify/e2e/SpotifyIconTest.java

```diff
@@ -0,0 +1,51 @@
+package com.qalabs.seleniumbasics.spotify.e2e;
+
+import com.qalabs.javabasics.spotify.pages.SpotifyHelpPage;
+import com.qalabs.javabasics.spotify.pages.SpotifyHomePage;
+import com.qalabs.seleniumbasics.spotify.BaseTest;
+import com.qalabs.seleniumbasics.spotify.utilities.PropertyReader;
+import org.testng.Assert;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+public class SpotifyIconTest extends BaseTest {
+    private String spotifyUrl, helpUrl;
+    private PropertyReader propertyReader = new PropertyReader();
+    private SpotifyHomePage spotifyHomePage;
+    private SpotifyHelpPage spotifyHelpPage;
+
+    @BeforeTest
+    public void initSetup() {
+        spotifyUrl = propertyReader.getProperty("credentials.properties", "URL_WEBSITE");
```

Comment:
These values are not user credentials. Create a new _properties_ file for these kinds of values